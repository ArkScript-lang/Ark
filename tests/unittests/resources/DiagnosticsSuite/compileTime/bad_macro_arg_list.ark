($ b ($symcat c))
(print b)
