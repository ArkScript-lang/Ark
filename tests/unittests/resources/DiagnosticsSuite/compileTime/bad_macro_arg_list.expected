At ( @ 1:6
    1 | ($ b ($symcat c))
      |      ^
    2 | (print b)
    3 |
        Expected a body while defined macro `b'. If you were trying to define a macro based on a function call, try wrapping it inside a begin node: ($ b { ($symcat c) }).
        Without the begin node, '($symcat c)' is seen as an argument list.
